nixie/unittests/test_verbose.py

```
@@ -83,3 +120,76 @@ async def fake_wait_for_proc(
+async def test_render_block_logs_unexpected_exception(
+    monkeypatch: pytest.MonkeyPatch,
+    tmp_path: Path,
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    cfg_path = tmp_path / "cfg.json"
+    cfg_path.write_text("{}")
+    semaphore = asyncio.Semaphore(1)
+    path = tmp_path / "doc.md"
+
+    async def raise_exception(*_args: object, **_kwargs: object) -> None:
+        raise Exception("uh oh")
```

Ruff (0.12.2)

Create your own exception

(TRY002)